src/components/modal/modal.tsx

```diff
@@ -6,7 +6,9 @@ import { ComponentInterface, Component, Host, h, Prop, Event, EventEmitter } fro
   shadow: true,
 })
 export class Modal implements ComponentInterface {
+  /** identify if modal is visible or not */
   @Prop() isVisible: boolean = true;
```

Comment:
You can just write the value instead put also the type

Reply:
I didn't write the code of this component, I also added the comments